contracts/run_snarknode.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+# built with `docker build --tag aleph-node:snarknet -f ./docker/Dockerfile .`
+export NODE_IMAGE=aleph-node:snarknet
+
+# key derived from "//0"
+export NODE_ID=5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
+# Alice well known key
+export ALICE=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+mkdir -p docker/data/
+
+# generate chainspec
+docker run --rm -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e RUST_LOG=debug "${NODE_IMAGE}" -c \
+       "aleph-node bootstrap-chain --base-path /data --account-ids $NODE_ID --faucet-account-id $ALICE --sudo-account-id $NODE_ID --chain-id a0smnet --token-symbol SZERO --chain-name 'Aleph Zero Smarknet' > /data/chainspec.smarknet.json"
```

Comment:
once you rebase on top of my recent commit, please add a second account for `//1`, so that it has money for submitting some transactions

Reply:
so there are basically two accounts with funds here /Alice and //0, will that do?